akka-cluster/src/main/scala/akka/cluster/ClusterDaemon.scala

```diff
@@ -1143,6 +1143,14 @@ private[cluster] class ClusterCoreDaemon(publisher: ActorRef) extends Actor with
       }
 
       publish(latestGossip)
+    } else {
```

Comment:
ok, pruning when there are no other changes, sounds good for a low priority task